doc/lua_api.txt

```diff
@@ -5935,9 +5937,11 @@ Player properties need to be saved manually.
     {
         hp_max = 1,
         -- For players only. Defaults to `minetest.PLAYER_MAX_HP_DEFAULT`.
+		-- Maximal value is 65535 (2^16 - 1)
```

Comment:
Please no extra indent, for consistency with other multiline comments.

Reply:
For these indented comments, have you used tabs instead of spaces, or spaces instead of tabs? Indentation looks weird.